### ISSUE TITLE
[Blazor] Cleanup feeds listed in the Blazor templates

### ISSUE THREAD

**Issue by pranavkm** [aspnet/AspNetCore#7813]
All of the packages that the templates need should be available on nuget.org. We shouldn't need to reference aspnetcore-dev or blazor-dev

e.g. https://github.com/aspnet/AspNetCore/blob/master/src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/BlazorHosted-CSharp.Server.csproj#L6-L7

**Comment by pranavkm** [comment]
cc @SteveSandersonMS.

**Comment by SteveSandersonMS** [comment]
We do need the dev feeds if we are going to continue publishing the nightly VSIXes.

The `Condition="'$(IncludeCustomRestoreSourcesSymbol)'=='true'"` is meant to auto-remove these from non-nightly builds (i.e., actual releases), but this hasn't been working properly since the mondo-repo move. I'm not sure if there is still any MSBuild property that knows whether a given build is "a proper release" or not.

**Comment by pranavkm** [comment]
Our CI system doesn't really have a notion of nightlies versus official builds. Do we still plan on doing fixups to the VSIX \ templates as part of Blazor releases? Perhaps we could do this change here. Alternatively documenting that you need to manually reference the feeds if you were to try out our nightlies seems fairly reasonable. I think as of today, you still need to reference the feed to test out AspNetCore's nightly builds, so this doesn't seem too much to ask of users. Thoughts?

**Comment by pranavkm** [comment]
Moving this out to 3.0 RTM. We can figure this out before we ship the RTM release.

**Comment by SteveSandersonMS** [comment]
> Alternatively documenting that you need to manually reference the feeds if you were to try out our nightlies seems fairly reasonable.

I'm not totally convinced :) If it fails out of the box, a large percentage of people will just conclude it's broken or low-quality and will give up straight away. It's especially nonobvious how to "reference the feeds" in non-VS cases, too.

> I think as of today, you still need to reference the feed to test out AspNetCore's nightly builds, so this doesn't seem too much to ask of users.

The SDK builds in a default reference to the ASP.NET Core dev feed, but *not* to the Blazor one. So it would be a new and different thing we are asking users to do.

In summary, if we can't differentiate between "nightly builds" and "actual preview releases", my preference would be to leave the feed references there in both cases. The other option of removing it only from the `release/preview-3` branch seems better still.

**Comment by pranavkm** [comment]
> my preference would be to leave the feed references there in both cases

👍 . So I suppose there's nothing to do here. Closing this. We can revisit these closer to an RTM release.
